backend/controllers/retentionReasons.js

```diff
@@ -11,25 +10,23 @@ const retentionReasonsFactory = (elite2Api, dataComplianceApi, logError) => {
 
   const getOffenderUrl = offenderNo => `${dpsUrl}offenders/${offenderNo}`
 
-  const formatDate = date => date && formatTimestampToDate(date)
-
-  const formatDateTime = dateTime => dateTime && formatTimestampToDateTime(dateTime)
-
   const getFormAction = offenderNo => `/offenders/${offenderNo}/retention-reasons`
 
   const getLastUpdate = existingRecord =>
-    existingRecord && {
-      version: existingRecord.etag,
-      timestamp: formatDateTime(existingRecord.modifiedDateTime),
-      user: existingRecord.userId,
-    }
+    existingRecord
+      ? {
+          version: existingRecord.etag,
+          timestamp: formatTimestampToDateTime(existingRecord.modifiedDateTime),
+          user: existingRecord.userId,
+        }
+      : {}
```

Comment:
an option here would have been to use destructuring: 
```js
 (existingRecord = {}) => ({ ...existingRecord, timestamp: formatDateTime(existingRecord.modifiedDateTime) })
```
But its kind of nice seeing the structure of the object anyway..

Reply:
Nice!